exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/database/MemoryDb.java

```diff
@@ -78,7 +78,8 @@ public Fork createFork(Cleaner cleaner) {
    * are closed and cannot be used anymore. Any subsequent operations on these objects will result
    * in {@link IllegalStateException}.
    *
-   * <p>TODO(@bogdanov): If the fork cannot be applied to the database …
+   * <p>If the fork cannot be applied to the database {@link RuntimeException} is
```

Comment:
I am thinking of any realistic scenarious that might trigger this to understand how to document it and if we need to translate RuntimeException into anything more appropriate. Fork from other database?

Reply:
Merging fork from other database is definitely possible (however, I can't say anything about the result of such action). I think `RuntimeException` is good here given low possibility of any errors and unability to recover from them.